### DISCUSSION TITLE
Question regarding dependency of java-11-openjdk-headless through javapackages-tools

### DISCUSSION THREAD

**Discussion by Macleykun** [candlepin/candlepin#4891] [Q&A] [answered]
Hello!

We use foreman katello on RHEL and i noticed that openjdk 11 is reaching EOL next week.
I saw that if i would remove that dependency that the main application would be removed as candlepin relies on javapackages-tools.

I wanted to ask if it's known that using that package as a dependency requires to run a soon EOL java version. If i should rather ask the maintainers of [javapackages-tools](https://github.com/fedora-java/javapackages) do let me know! Then i'll contact them and refer to this issue/the one below aswell!

Me Asking the foreman community about it: https://community.theforeman.org/t/question-regarding-java-11-openjdk-headless/39950/4

**Comment by nikosmoum** [comment]
Hi!, this is news to us; thanks for bringing it to our attention. If you can also let the javapackages-tools folks it would be great. If they can't get an update for this soon enough, we'll try to find a solution on our end (javapackages-tools is only really used during the initial schema update, and not by the running tomcat/candlepin service, so hopefully we can work around it).

**Reply by nikosmoum** [reply, marked as answer]
I've created an issue on their side: https://github.com/fedora-java/javapackages/issues/134 and I'll look into working around it for now.

**Reply by Macleykun** [reply]
For RHEL users, openjdk will automatically be switch from openjdk 11 to 16 when rhel 9.5/9.6 comes! Assuming within the context of using candlepin standalone (or with foreman).
thank you for asking futher!
